test/index.html

```diff
@@ -2,12 +2,12 @@
 <html>
   <head>
     <title>integrations tests</title>
-    <link rel="stylesheet" href="/node_modules/mocha/mocha.css">
+    <link rel="stylesheet" href="../node_modules/mocha/mocha.css">
```

Comment:
I changed this so that I could use my IDE, (PyCharm) to host the HTML files and test, (instead of Express.)

I've verified that `make test-browser` still works correctly.